### PR TITLE
init pypi github actions config

### PR DESCRIPTION
**Pull request by robert-bryson**
# Pull Request

Related to https://github.com/GSA/data.gov/issues/4544

## About

Related to https://github.com/GSA/datagov-harvester/pull/6

## PR TASKS

- [ ] The actual code changes.
- [ ] Tests written and passed.
- [ ] Any changes to docs?
- [ ] Bumped version number in [pyproject.toml](https://github.com/GSA/datagov-harvesting-logic/blob/8078c5b9f015ca7fb8a142e4e4b2e22ad2fd49b1/pyproject.toml#L3) (also checked on [PyPi](https://pypi.org/project/ckanext-geodatagov/#history)).

### CODE REVIEW

**Comment by rshewitt** [comment]
catalog proxy on dev needs to be cherry picked back to allow package creation. restarting the actions in a moment should be good.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    author="Data.gov",
+    author_email="datagovhelp@gsa.gov",
+    url="https://github.com/GSA/datagov-harvesting-logic/",
+    license="Public Domain",
+    packages=find_packages(exclude=["ez_setup", "examples", "tests"]),
+    namespace_packages=["harvester"],
+    include_package_data=True,
+    zip_safe=False,
+    install_requires=[
+        # TODO anything?
```
Comment (rshewitt) [inline setup.py]: i'm not familiar with this option so i can't say whether something already accounted for in our pyproject.toml would be better placed here. I think we can probably leave this blank though.

File: setup.py
Comment (rshewitt) [inline setup.py:1]: can't we use the pyproject.toml for packaging instead of this?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ],
+    setup_requires=["wheel"],
+    entry_points="""
+        # TODO anything?
```
Comment (rshewitt) [inline setup.py]: seems like an advanced feature we don't need at least for now

File: setup.py (hunk 1 above)
Comment (btylerburton) [reply setup.py]: thanks for noting that @rshewitt 
can we follow the new standard?
https://www.reddit.com/r/learnpython/comments/yqq551/pyprojecttoml_setupcfg_setuppy_whats_the/

**Review by jbrown-xentity — DISMISSED** [review]
Looks great!

**Review by rshewitt — APPROVED** [review]

**Pull request merged**